File: src/main/java/Device.java
class Device {

  private final String name;
  private final MACAddress macAddress;

  public Device(final String name, final MACAddress macAddress) {
    this.name = name;
    this.macAddress = macAddress;
  }

  public MACAddress getMacAddress() {
    return macAddress;
  }

  @Override
  public String toString() {
    return name + " (" + macAddress + ")";
  }
}

File: src/main/java/com/adamkoch/iptables/objects/Device.kt
package com.adamkoch.iptables.objects

import com.adamkoch.iptables.objects.MacAddress

/**
 * Represents a network-enabled device such as a computer, TV, gaming system, etc.
 *
 * @author aakoch
 * @since 0.1.0
 */
class Device(val name: String, val macAddress: MacAddress) {
    override fun toString(): String {
        return "$name ($macAddress)"
    }
}